frontend/src/scenes/sceneLogic.ts

```diff
@@ -63,8 +65,6 @@ export const scenes: Record<Scene, () => any> = {
     [Scene.ErrorNetwork]: () => ({ default: preloadedScenes[Scene.ErrorNetwork].component }),
     [Scene.Dashboards]: () => import(/* webpackChunkName: 'dashboards' */ './dashboard/Dashboards'),
     [Scene.Dashboard]: () => import(/* webpackChunkName: 'dashboard' */ './dashboard/Dashboard'),
-    [Scene.DashboardInsight]: () =>
```

Comment:
This was something I had built previously that I don't think we'll need for now, I've left some files up in the folder for it because I will probably make use of the dashboard insight header code from that folder in my next PR. And then I'll clean it up fully~